test/unit/LivepeerETHTicketBroker.js

```diff
@@ -304,6 +306,30 @@ contract("LivepeerETHTicketBroker", accounts => {
 
                 assert.equal(events.length, 0)
             })
+
+            describe("withdrawTransfer", () => {
```

Comment:
Shouldn't this be nested under a top level describe block for `withdraw` and not under the describe block for `penaltyEscrowSlash` (which itself is nested under `redeemWinningTicket`)?

Reply:
yes, I think it should be good in the latest commit!